hazelcast/src/test/java/com/hazelcast/partition/impl/MigrationQueueTest.java

```diff
@@ -0,0 +1,77 @@
+package com.hazelcast.partition.impl;
+
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+@Category(QuickTest.class)
+public class MigrationQueueTest {
+
+    private final MigrationQueue migrationQueue = new MigrationQueue();
+
+    @Test
+    public void test_migrationTaskCount_incremented_onMigrateTask() {
```

Comment:
Do you think it is wise if we allow everybody to have their own preference when writing a test? Imho it makes code harder to understand. 

We have a test practices page on the wiki where we have a naming approach. If you don't like it and want to improve it, I'm all ear. But imho we should do this consistently using the whole system. I